test/features/specification/monitoring.js

```diff
@@ -53,9 +46,21 @@ function constructTests(spec, defParams) {
             }
             p['x-amples'].forEach(function(ex) {
                 ex.request = ex.request || {};
+                if (ex.request.params && ex.request.params.domain !== options.domain) {
+                    return;
+                }
+
+                // This is a hack. We're running tests again parsoid instance in labs
```

Comment:
Yep, don't like that too.. I'm explaining the need for this in the pr description, and if you have any suggestions how to do it better - I'm both hands up for doing this other way.